scripts/variantstore/wdl/GvsExtractCallset.wdl

```diff
@@ -24,8 +24,7 @@ workflow GvsExtractCallset {
     String drop_state = "NONE"
 
     File interval_list = "gs://gcp-public-data--broad-references/hg38/v0/wgs_calling_regions.hg38.noCentromeres.noTelomeres.interval_list"
-    Boolean use_interval_weights = true
-    File interval_weights_bed = "gs://broad-public-datasets/gvs/weights/gvs_vet_weights_1kb.bed"
+    File interval_weights_bed = "gs://gvs_quickstart_storage/weights/gvs_full_vet_weights_1kb_padded_orig.bed"
```

Comment:
Could we get these put up into gs://broad-public-datasets?

Reply:
I don't have permission to do so, and honestly?  I'm not sure it's the correct location given that this file is exclusively used by gvs.  I don't think "gvs_quickstart_storage" looks like to be the best place for the file gvs needs to provide for public usage purely do to the naming, but it's the best place we currently have.